packages/passport/sdk/src/storage/IndexedDbStorage.ts

```diff
@@ -0,0 +1,37 @@
+import { AsyncStorage } from 'oidc-client-ts';
+import localForage from 'localforage';
+
+export class IndexedDbStorage implements AsyncStorage {
```

Comment:
Can we update the name to indicate that this is a `localForage` wrapper please? Maybe `LocalForageAsyncStorage`?

Reply:
Following the same train of thought, can we either:

- include IndexedDB in the name, OR
- pass the driver (and the name) as an argument to the constructor